plugins/flytekit-ray/flytekitplugins/ray/models.py

```diff
@@ -178,9 +192,13 @@ def __init__(
         self,
         ray_cluster: RayCluster,
         runtime_env: typing.Optional[str],
+        ttl_seconds_after_finished: typing.Optional[int] = None,
```

Comment:
this is a no-op if `shutdown_after_job_finishes` is set to False, right?

Reply:
Yes.